content/docs/5.1/deploy/benchmark/benchmark-instructions.md

```diff
@@ -0,0 +1,155 @@
+| :------ | :------------: | :------------------ |
+| sshpass | 1.06 or above  | remote control      |
+| TiUP    | 0.6.2 or above | depoly TiKV cluster |
+
+For the component node:
+
+| package |     version     |          comment           |
+| :------ | :-------------: | :------------------------: |
+| sshpass |  1.06 or above  |       remote control       |
+| numa    | 2.0.12 or above | memory allocation strategy |
+| tar     | no requirements |           unzip            |
+
+For the YCSB node:
+
+| package                                       |     version     |  comment  |
+| :-------------------------------------------- | :-------------: | :-------: |
+| [go-ycsb](https://github.com/pingcap/go-ycsb) | no requirements | benchmark |
+
+
+{{< info >}}
+You can install [TiUP](https://github.com/pingcap/tiup) as described in [TiKV in 5 Minutes](../../tikv-in-5-minutes).
+{{< /info >}}
+
+## Step 2. Deploy a cluster
+
+1. We could use the following topology to deploy our benchmark cluster via `tiup cluster`. **For more information, see [Production Deployment](../../install/production)**
+```yaml
+global:
+  user: "tikv"
+  ssh_port: 22
+  deploy_dir: "/tikv-deploy"
+  data_dir: "/tikv-data"
+server_configs: {}
+pd_servers:
+  - host: 10.0.1.1
+tikv_servers:
+  - host: 10.0.1.2
+  - host: 10.0.1.3
+  - host: 10.0.1.4
+monitoring_servers:
+  - host: 10.0.1.5
+grafana_servers:
+  - host: 10.0.1.6
+```
+
+2. Save the content above as `topology.yaml`:
+```sh
+tiup cluster deploy [cluster-name] [cluster-version] topology.yaml
+```
+
+3. Start the cluster:
+```sh
+tiup cluster start [cluster-name]
+```
+
+4. You can inspect the cluster information:
+```sh
+tiup cluster display [cluster-name]
+```
+
+## Step 3. Run a YCSB workload
+
+YCSB has six kinds of workloads whose main difference are the portion of different operations. All six workloads have a data set which is similar. Workloads D and E insert records during the test run. Thus, to keep the database size consistent, we recommend the following sequence:
+1. Load the database, using workload A's parameter file (workloads/workloada) and the "-load" switch to the client.
+2. Run workload A (using workloads/workloada and "-t") for a variety of throughputs.
+3. Run workload B (using workloads/workloadb and "-t") for a variety of throughputs.
+4. Run workload C (using workloads/workloadc and "-t") for a variety of throughputs.
+5. Run workload F (using workloads/workloadf and "-t") for a variety of throughputs.
+6. Run workload D (using workloads/workloadd and "-t") for a variety of throughputs. This workload inserts records, increasing the size of the database.
+7. Delete the data in the database.
+8. Reload the database, using workload E's parameter file (workloads/workloade) and the "-load switch to the client.
+9. Run workload E (using workloads/workloade and "-t") for a variety of throughputs. This workload inserts records, increasing the size of the database.
```

Comment:
would you please give the shell command?

Reply:
what's the differences between workload A-E?
how can users choose from workload A-E?

Reply:
Normally, users should run all the workload and check the workload they are interested in.

Reply:
what's the differences between workload A-E?